.github/workflows/deploy_branch_preview.yml

```diff
@@ -60,8 +65,21 @@ jobs:
             ${{ runner.os }}-
       - name: Install Packages
         run: npm install
+      - name: Get Latest Release
+        id: latestRelease
+        run: |
+          RELEASE_TAG=$(echo $(git describe --tags --abbrev=0))
+          echo "::set-output name=release::$RELEASE_TAG"
+      - name: Echo Version Details
+        env:
+          LATEST_RELEASE: ${{steps.latestRelease.outputs.release}}
```

Comment:
Since we set it as an output in the previous step, do we need this `env`?

Reply:
erm im using bash's param substitution to remove the `prod-` which only works if its a bash env variable so I'm injecting that. alternatively, i can shift that into the run script